source/electrical_heating/src/electrical_heating/domain/calculation.py

```diff
@@ -49,196 +52,335 @@ def execute_core_logic(
     child_metering_points: DataFrame,
     time_zone: str,
 ) -> DataFrame:
-    child_metering_points = child_metering_points.where(
+    child_points = child_metering_points.where(
```

Comment:
We also talk about points in time series. So this is ambiguous to me. I prefer the old name.
```suggestion
    child_metering_points = child_metering_points.where(
```

Reply:
This comment applies to all variables referring to metering points (as opposed to time series points).